src/App/Datums.h

```diff
@@ -20,50 +22,182 @@
  *                                                                         *
  ***************************************************************************/
 
-#ifndef ORIGINFEATURE_H
-#define ORIGINFEATURE_H
+#ifndef DATUMS_H
+#define DATUMS_H
 
```

Comment:
This belongs to very first commit of this PR.

Reply:
fixed in the other pr.